excel.go

```diff
@@ -225,6 +230,36 @@ func copyCell(f *excelize.File, automatedSheetName string, srcX, srcY, destX, de
 	return nil
 }
 
+func sortRows(f *excelize.File, input *Input, sheetname string) error {
+	rows, err := f.GetRows(sheetname)
+	if err != nil {
+		return err
+	}
+
+	colUser := input.AutomatedSheetColNameToIndex[ColUser]
+	colPassword := input.AutomatedSheetColNameToIndex[ColPassword]
+	colPrevious := input.AutomatedSheetColNameToIndex[ColPrevious]
+	colTimestamp := input.AutomatedSheetColNameToIndex[ColTimestamp]
+
+	sort.Slice(rows[input.RowOffset:], func(i, j int) bool {
+		return rows[input.RowOffset+i][colUser] < rows[input.RowOffset+j][colUser]
+	})
+	// write sorted rows to automatedSheet
+	for idx, r := range rows[input.RowOffset:] {
+		cellName := fmt.Sprintf("A%d", 2+idx)
+		err = f.SetSheetRow(sheetname, cellName, &[]string{r[colUser], r[colPassword], r[colPrevious], r[colTimestamp]})
```

Comment:
doesn't this line mean you're assuming the order of the columns again?

Reply:
Yes. I get it now. I just want to write r[0], r[1], r[2], r[3] starting from col A.